### PR TITLE
create Round model

### PR DESCRIPTION
**Pull request by SergeyBasharkin**

### CODE REVIEW

File: app/models/question.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class Question < ActiveRecord::Base
+  belongs_to :round
```
Comment (vladimirbazhanov) [inline app/models/question.rb]: I think here we need to have other model "RoundQuestion".

Round will have 4 RoundQuestions

RoundQuestion will be simple model with round_id and question_id columns.

When user starts new round we will find 4 'questions' and create 4 'round question' for new created game.

Looks good?

File: app/models/round.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Round < ActiveRecord::Base
+  has_many :questions
```
Comment (vladimirbazhanov) [inline app/models/round.rb]: see prev comment, I think this need to be removed

File: spec/factories/games.rb [hunk 3]
```diff
@@ -1,7 +1,16 @@
 FactoryGirl.define do
   factory :game do
+    id 1
     player_1 1
     player_2 nil
     state "search"
+    factory :game_with_3_rounds do
```
Comment (vladimirbazhanov) [inline spec/factories/games.rb]: I think that factory :game always should be built with 3 rounds. We will never want to build 'game' without rounds

File: app/models/question.rb (hunk 1 above)
Comment (SergeyBasharkin) [inline app/models/question.rb]: Ok, I'll remake it😃

File: db/schema.rb [hunk 4]
```diff
@@ -31,6 +31,24 @@
   add_index "active_admin_comments", ["namespace"], name: "index_active_admin_comments_on_namespace", using: :btree
   add_index "active_admin_comments", ["resource_type", "resource_id"], name: "index_active_admin_comments_on_resource_type_and_resource_id", using: :btree
 
+  create_table "admin_users", force: :cascade do |t|
```
Comment (vladimirbazhanov) [inline db/schema.rb]: this is old table as I know, can be removed from schema

**Comment by vladimirbazhanov** [comment]
👍

**Pull request merged**